common/download_iso_and_transfer.yml

```diff
@@ -33,7 +33,7 @@
   debug: var=transferred_install_iso
 
 - name: Copy downloaded OS installation ISO file to ESXi host
-  include_tasks: ../../common/transfer_file_remote.yml
+  include_tasks: transfer_file_remote.yml
```

Comment:
I think here can replace with esxi_upload_datastore_file.yml.

Reply:
Yeah, right, changed it to esxi_upload_datastore_file.yml. Thanks.